ui/lib/apps/Statement/utils/useStatement.ts

```diff
@@ -125,7 +125,7 @@ export default function useStatement(
     }
 
     queryStatementList()
-  }, [queryOptions, allTimeRanges, validTimeRange, refreshTimes])
+  }, [queryOptions, allTimeRanges, validTimeRange])
```

Comment:
the `refreshTimes` is already depended by another `useEffect()` which will update allTimeRanges and validTimaRange.